dataprofiler/profilers/base_column_profilers.py

```diff
@@ -247,6 +247,34 @@ def report(self, remove_disabled_flag: bool = False) -> dict:
         """
         raise NotImplementedError()
 
+    @classmethod
+    def json_to_object(cls, data) -> BaseColumnProfiler:
+        """
+        Parse attribute from json dictionary into self.
+
+        :param data: dictionary with attributes and values.
+        :type data: dict[string, Any]
+
+        :return: Profiler with attributes populated.
+        :rtype: BaseColumnProfiler
+        """
+        profile = cls(data["name"])
+
+        time_vals = data.pop("times")
+        setattr(profile, "times", defaultdict(float, time_vals))
+
+        for attr, value in data.items():
+            if "__calculations" in attr:
+                for metric, function in value.items():
+                    if not hasattr(profile, function):
+                        raise AttributeError(
+                            f"Object {type(profile)} has no attribute {function}."
+                        )
+                    value[metric] = getattr(profile, function)
```

Comment:
Long term I think will need to pull this out into a function bc i think it will be reused by other classes which don't have this as the base.

Reply:
yeah I agree -- refactor into a function (away from a `@classmethod`)